src/components/EnvironmentOverride/SecretOverrides.tsx

```diff
@@ -325,14 +342,19 @@ export function OverrideSecretForm({ name, appChartRef, toggleCollapse }) {
                 })
                 setSecretData(json)
             }
+            if(configData[0].esoSecretData.esoData) {
+                setEsoData(configData[0].esoSecretData?.esoData)
+                setSecretStore(configData[0].esoSecretData?.secretStore)
```

Comment:
keep optional check in if condition and remove from here

Reply:
Done